db/migrate/20161003203532_add_lyrics_column_to_songs.rb

```diff
@@ -0,0 +1,6 @@
+# this migration will fail on databases that already contain songs
+class AddLyricsColumnToSongs < ActiveRecord::Migration
+  def change
+    add_column :songs, :lyrics, :text, null: false, after: :chord_sheet
```

Comment:
If we're searching against this now then we'll want an index, gin type I think. Also should we remove the index from chord_sheet?